src/components/style_dropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import "./dropdown_styles.css";
+import { BiPlus } from "react-icons/bi";
+import { IoCheckmarkSharp } from "react-icons/io5";
+import { useState } from "react";
+
+export default function Styledropdown() {
+  const [icon, setIcon] = useState(<BiPlus size={18} />);
+  function Showhide() {
+    const click = document.getElementById("myDropdown");
+    if (click?.style.display === "none") {
+      click.style.display = "block";
+      setIcon(<IoCheckmarkSharp size={18} />);
+    } else {
+      click!.style.display = "none";
+      setIcon(<BiPlus size={18} />);
+    }
+  }
+  return (
+    <div className="dropdown">
+      <button type="button" onClick={Showhide} className="dropbtn">
+        <span id="Change"> {icon} </span>
+        <span className="C1"> Style </span>
+      </button>
+      <div id="myDropdown" className="dropdown-content">
+        <div className="font">
+          <ul>
+            <li>
+              <span className="items">
```

Comment:
the items in the list will probably have to be dynamically listed as we have more types of items but for now, this should be good.